### PR TITLE
Add xml command "GetFanSpeed"

### PR DESCRIPTION
**Pull request by flubshi**

### CODE REVIEW

**Comment by flubshi** [comment]
pre-commit.ci autofix

File: deebot_client/events/fan_speed.py [hunk 1]
```diff
@@ -17,6 +17,8 @@ class FanSpeedLevel(DisplayNameIntEnum):
     NORMAL = 0
     MAX = 1
     MAX_PLUS = 2
+    STANDARD = 3
+    STRONG = 4
```
Comment (edenhaus) [inline deebot_client/events/fan_speed.py:21]:
```suggestion
```

The JSON-based vacuums use the numbers to map to the correct value. Therefore we cannot introduce new ones in this enum.

Also, let's reuse the values already defined here. I would suggest we map standard to normal and strong to max. What do you think?

Comment (flubshi) [reply deebot_client/events/fan_speed.py]: I agree. It makes sense to map it to existing values. I updated this PR accordingly

**Comment by flubshi** [comment]
pre-commit.ci autofix

**Comment by flubshi** [comment]
pre-commit.ci autofix

**Review by edenhaus — APPROVED** [review]
Thanks @flubshi :+1:

**Pull request merged**